templates/_layout.html

```diff
@@ -24,39 +24,23 @@
 
     {% block content %}{% endblock %}
 
-    <footer class="p-footer u-no-margin--top">
-      <div class="row">
-        <div class="col-9">
-          &copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
-          <nav>
-            <ul class="p-footer__links">
-                <li class="p-footer__item"><a class="p-footer__link" href="http://www.ubuntu.com/legal">Legal information</a></li>
-                <li class="p-footer__item"><a class="p-footer__link" href="https://github.com/canonical-websites/snapcraft.io/issues/new">Report a bug on this site</a></li>
-                <li class="p-footer__item"><a class="p-footer__link" href="http://status.snapcraft.io/">Service status</a></li>
-            </ul>
-            <span class="u-off-screen">
-              <a href="#">Go to the top of the page</a>
-            </span>
-          </nav>
-        </div>
-        <div class="col-3">
-          <h3 class="p-muted-heading">Follow us</h3>
-          <ul class="p-inline-list--compact">
-            <li class="p-inline-list__item">
-              <a href="https://twitter.com/snapcraftio" class="p-social-icon--twitter">Share on Twitter</a>
-            </li>
-            <li class="p-inline-list__item">
-              <a href="https://plus.google.com/+SnapcraftIo" class="p-social-icon--google">Share on Google plus</a>
-            </li>
-            <li class="p-inline-list__item">
-              <a href="https://www.facebook.com/snapcraftio" class="p-social-icon--facebook">Share on Facebook</a>
-            </li>
-            <li class="p-inline-list__item">
-              <a href="https://www.youtube.com/snapcraftio" class="p-social-icon--youtube">Share on YouTube</a>
-            </li>
-          </ul>
+    <div class="p-strip--light u-no-padding">
+      <footer class="p-footer u-no-margin--top">
+        <div class="row">
+          <div class="col-12">
+            Directory &copy; 2017 Canonical Ltd.
+            <nav class="u-no-margin">
```

Comment:
To avoid unexpected bottom margin (caused likely by https://github.com/vanilla-framework/vanilla-framework/issues/1338) additional `u-no-margin` is needed.